gradle.properties

```diff
@@ -11,7 +11,12 @@
 
 org.gradle.warning.mode=none
 org.gradle.parallel=true
-org.gradle.jvmargs=-Xmx3g -XX:+HeapDumpOnOutOfMemoryError -Xss2m
+org.gradle.jvmargs=-Xmx3g -XX:+HeapDumpOnOutOfMemoryError -Xss2m \
+    --add-exports jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED \
```

Comment:
I am wondering why these exports are needed? AFAIK OpenSearch does not use Java Compiler API, do you have an example at hand? Thank you.

Reply:
Ah ... thank you @martin-gaievski , `spotless` again ...